src/data-pipeline/spark-etl/src/main/java/software/aws/solution/clickstream/TransformerV2.java

```diff
@@ -108,6 +107,11 @@ public final class TransformerV2 {
     public static final String EVENT_VALUE_IN_USD = "event_value_in_usd";
     public static final String DEVICE_ID_LIST = "device_id_list";
     public static final String APP_ID = "app_id";
+    public static final String REFERRER = "_referrer";
+    public static final String PROP_PAGE_REFERRER = "_page" + REFERRER;
+    public static final String REFERER = "_referer";
+    public static final String COL_PAGE_REFERER = "_page" + REFERER;
```

Comment:
why renaming it to `_page_referer` in glue table?

`referrer` is the correct word

Reply:
![image](https://github.com/awslabs/clickstream-analytics-on-aws/assets/23469402/f0f2149b-67d4-4904-9a1a-c631d6480f37)

referer is still correct.